Are there types available?

Hello guys! I am trying to use this analytics script in an existing typescript environment. I tested the vanilla js and it worked pretty well, now I'd like to integrate the tool in my existing typescript application.
My question is: are there any types available or is there documentation available for using this with TypeScript? 
_PS: I tried searching in the available documentation in github as well for existing types in npm / [TypeSearch](https://microsoft.github.io/TypeSearch/), nothing there._

```
node_modules/analytics/lib/types.d.ts:496:21 - error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.

496 export const init = analytics;
                        ~~~~~~~~~

node_modules/analytics/lib/types.d.ts:498:26 - error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.

498 export const Analytics = analytics;
                             ~~~~~~~~~

node_modules/analytics/lib/types.d.ts:500:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

500 export = analytics;
    ~~~~~~~~~~~~~~~~~~~
```
This is what I get when using `npm install analytics --save` and afterwards `import Analytics from 'analytics';` in TypeScript. Is this intended? @DavidWells

--- a/packages/analytics-core/package.json
+++ b/packages/analytics-core/package.json
@@ -33,7 +33,7 @@
     "test": "ava -v",
     "test:watch": "ava -v --watch",
     "clean": "rimraf lib dist && mkdirp lib dist",
-    "prebuild": "npm run clean",
+    "prebuild": "npm run clean && npm run types",
     "types": "../../node_modules/.bin/jsdoc -t ../../node_modules/tsd-jsdoc/dist -r ./src/ -d temp-types && node scripts/types.js",
     "build": "node ../../scripts/build/index.js",
     "postbuild": "npm run minify-dist && node ./scripts/postbuild.js",

